offer-manager-domain/src/main/groovy/life/qbic/business/offers/Offer.groovy

```diff
@@ -261,28 +261,35 @@ class Offer {
      * @return The net offer price including overheads
      */
     double getTotalNetPriceWithOverheads() {
-        return calculateNetPrice() - getTotalDiscountAmount() + getOverheadSum()
+        return (calculateNetPrice() + getOverheadSum()) as BigDecimal
     }
 
     /**
      * The overhead price amount of all service items without VAT.
      *
      * @return The calculated overhead amount of the selected items.
      */
-    double getOverheadSum() {
+    BigDecimal getOverheadSum() {
         if (items.empty) {
             return 0
         }
-        return items.sum {calculateItemOverhead(it)} as double
+        return items.sum {calculateItemOverhead(it)} as BigDecimal
```

Comment:
I do not know how the casting to BigDecimal is handled by groovy.
In case this calls new BigDecimal(yourDoubleValue) this is not the desired behaviour (as stated by the BigDecimal documentation).
You should use `new BigDecimal(double.toString())` or `BigDecimal.valueOf(double)`